src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
     images::Vector{TiledImage}
     vp::VariationalParams{NumType}
     tile_source_map::Vector{Matrix{Vector{Int}}}
     patches::Matrix{SkyPatch}
     active_sources::Vector{Int}
+
+    # The number of standard deviations away below which a bivariate normal
+    # will be calculated.  Set to Inf to evaluate bivariate normals at all points.
+    num_allowed_sd::Float64
```

Comment:
I'm wary of creating setting that give developers the option of creating difficulty-to-track-down optimization bugs, but I do see the benefit. How much fast are the unit tests? Could we get the same benefit from more aggressive a prior trimming?

Another idea: I wonder if we could add some smoothing to a truncated Gaussian, that interpolates between the a few pixels on either side of the truncation boundary. Then we'd still have a continuous function, but finite support. We might then be able to set `num_allowed_sd` very aggressively, since the brightest pixels are usually way more informative than the farther out pixels.